sdk/core/core-rest-pipeline/src/nodeHttpClient.ts

```diff
@@ -309,6 +309,8 @@ function getBodyLength(body: RequestBodyType): number | null {
     return null;
   } else if (isArrayBuffer(body)) {
     return body.byteLength;
+  } else if (typeof body === "string") {
+    return Buffer.from(body).length;
```

Comment:
do you think it is worth it to factor the check into its own function similar to other branches checks?

Reply:
A string check is a very basic check. I don't think it needs a separate method for now.